### ISSUE TITLE
Incompatible with Java 8

### ISSUE THREAD

**Issue by pwouik** [vktec/redlog#3]
when entering /rlog pass tic (or something else)

redlog-1.16.4-0.2.0

playing fabric 1.16.5 fabric-api-0.29.4+1.16

happen even without others mods except fabric api

```
[17:39:17] [Server thread/ERROR]: Encountered an unexpected exception
java.lang.NoSuchMethodError: java.util.List.of(Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;)Ljava/util/List;
	at vktec.redlog.expr.Tokenizer.<clinit>(Tokenizer.java:11) ~[redlog-1.16.4-0.2.0(1).jar:?]
	at vktec.redlog.expr.RedExpr.<init>(RedExpr.java:10) ~[redlog-1.16.4-0.2.0(1).jar:?]
	at vktec.redlog.RedFilterRule.<init>(RedFilterRule.java:10) ~[redlog-1.16.4-0.2.0(1).jar:?]
	at vktec.redlog.RedlogMod.passRule(RedlogMod.java:41) ~[redlog-1.16.4-0.2.0(1).jar:?]
	at vktec.redlog.RedlogMod.lambda$onInitialize$1(RedlogMod.java:25) ~[redlog-1.16.4-0.2.0(1).jar:?]
	at vktec.redlog.RedlogMod$$Lambda$4356/176877731.run(Unknown Source) ~[?:?]
	at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:262) ~[brigadier-1.0.17.jar:?]
	at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:176) ~[brigadier-1.0.17.jar:?]
	at net.minecraft.class_2170.method_9249(class_2170.java:249) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_3244.method_14370(class_3244.java:1211) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_3244.method_31286(class_3244.java:1198) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_3244.method_12048(class_3244.java:1177) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_2797.method_12115(class_2797.java:36) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_2797.method_11054(class_2797.java:9) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_2600.method_11072(class_2600.java:21) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_2600$$Lambda$5694/1741045910.run(Unknown Source) ~[?:?]
	at net.minecraft.class_3738.run(class_3738.java:18) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_1255.method_18859(class_1255.java:144) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_4093.method_18859(class_4093.java:23) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.method_24306(MinecraftServer.java:761) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.method_18859(MinecraftServer.java:155) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_1255.method_16075(class_1255.java:118) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.method_20415(MinecraftServer.java:743) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.method_16075(MinecraftServer.java:737) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_1255.method_18857(class_1255.java:127) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.method_16208(MinecraftServer.java:722) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.handler$cak000$modifiedRunLoop(MinecraftServer.java:5319) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:651) [intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257) [intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer$$Lambda$5186/1978152868.run(Unknown Source) [intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_51]
[17:39:17] [Server thread/ERROR]: This crash report has been saved to: C:\Users\DELL\AppData\Roaming\.minecraft\crash-reports\crash-2021-01-25_17.39.17-server.txt
[17:39:17] [Server thread/INFO]: Stopping server
[17:39:17] [Server thread/INFO]: Saving players[17:39:17] [Server thread/ERROR]: Encountered an unexpected exception
java.lang.NoSuchMethodError: java.util.List.of(Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;)Ljava/util/List;
	at vktec.redlog.expr.Tokenizer.<clinit>(Tokenizer.java:11) ~[redlog-1.16.4-0.2.0(1).jar:?]
	at vktec.redlog.expr.RedExpr.<init>(RedExpr.java:10) ~[redlog-1.16.4-0.2.0(1).jar:?]
	at vktec.redlog.RedFilterRule.<init>(RedFilterRule.java:10) ~[redlog-1.16.4-0.2.0(1).jar:?]
	at vktec.redlog.RedlogMod.passRule(RedlogMod.java:41) ~[redlog-1.16.4-0.2.0(1).jar:?]
	at vktec.redlog.RedlogMod.lambda$onInitialize$1(RedlogMod.java:25) ~[redlog-1.16.4-0.2.0(1).jar:?]
	at vktec.redlog.RedlogMod$$Lambda$4356/176877731.run(Unknown Source) ~[?:?]
	at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:262) ~[brigadier-1.0.17.jar:?]
	at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:176) ~[brigadier-1.0.17.jar:?]
	at net.minecraft.class_2170.method_9249(class_2170.java:249) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_3244.method_14370(class_3244.java:1211) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_3244.method_31286(class_3244.java:1198) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_3244.method_12048(class_3244.java:1177) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_2797.method_12115(class_2797.java:36) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_2797.method_11054(class_2797.java:9) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_2600.method_11072(class_2600.java:21) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_2600$$Lambda$5694/1741045910.run(Unknown Source) ~[?:?]
	at net.minecraft.class_3738.run(class_3738.java:18) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_1255.method_18859(class_1255.java:144) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_4093.method_18859(class_4093.java:23) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.method_24306(MinecraftServer.java:761) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.method_18859(MinecraftServer.java:155) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_1255.method_16075(class_1255.java:118) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.method_20415(MinecraftServer.java:743) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.method_16075(MinecraftServer.java:737) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.class_1255.method_18857(class_1255.java:127) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.method_16208(MinecraftServer.java:722) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.handler$cak000$modifiedRunLoop(MinecraftServer.java:5319) ~[intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:651) [intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257) [intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer$$Lambda$5186/1978152868.run(Unknown Source) [intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_51]
[17:39:17] [Server thread/ERROR]: This crash report has been saved to: C:\Users\DELL\AppData\Roaming\.minecraft\crash-reports\crash-2021-01-25_17.39.17-server.txt
[17:39:17] [Server thread/INFO]: Stopping server
[17:39:17] [Server thread/INFO]: Saving players
```

**Comment by silversquirl** [comment]
Java 11 or higher is required for the current version of Redlog - I'll try to make it compatible with Java 8 in the next release.

**Comment by silversquirl** [comment]
Fixed in 86a11da
